.github/workflows/publish-nightly.yml

```diff
@@ -0,0 +1,121 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Based on Apache Arrow's java-nightly workflow
+# https://github.com/apache/arrow/blob/master/.github/workflows/java_nightly.yml
+name: Publish nightly artifacts
+
+on:
+  workflow_dispatch:
+    inputs:
+      keep:
+        description: Number of versions to keep.
+        required: false
+        default: 60
+  schedule:
+    - cron: "0 0 * * *"
+  push: # fixme
+    branches:
+      - 'seglo/publish-to-nightlies'
+
+permissions:
+  contents: read
+
+env: #fixme
+  GROUP_ID: 'com.typesafe.akka'
+
+jobs:
+  publish-nightly:
+    name: Publish nightly
+    runs-on: ubuntu-20.04
+    if: github.repository == 'apache/incubator-pekko'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up JDK 11
+        uses: olafurpg/setup-scala@v13
+        with:
+          java-version: adopt@1.11
+
+#      - name: Publish local
+#        run: |-
+#          sudo apt-get install graphviz
+#          # disable mima check until first pekko release is done
+#          # sbt +mimaReportBinaryIssues
+#          sbt +publishLocal
+      - name: Fake release
+        run: |-
+          mkdir -p $HOME/.ivy2/local/$GROUP_ID/pekko-actor-typed_2.13/0.0.0+${{ github.run_id }}-SNAPSHOT
+          touch $HOME/.ivy2/local/$GROUP_ID/pekko-actor-typed_2.13/0.0.0+${{ github.run_id }}-SNAPSHOT/release
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.4.0
+
+      - name: Sync from remote
+        uses: ./.github/actions/sync-nightlies
+        with:
+          switches: -avzh --update --delete --progress
+          local_path: repo
+          remote_path: ${{ secrets.NIGHTLIES_RSYNC_PATH }}/pekko/snapshots
```

Comment:
@seglo is the apache github organization setup with these secrets?

Reply:
@pjfanning Yes it should be. I have an open ticket with INFRA. https://issues.apache.org/jira/browse/INFRA-23895